src/main/kotlin/main.kt

```diff
@@ -1,12 +1,22 @@
-import reposfinder.ReposFinder
-import reposfinder.utils.tokenReader
+import reposfinder.config.SearchConfig
+import reposfinder.config.WorkConfig
+import reposfinder.logic.ReposFinder
+import reposfinder.utils.readList
+import reposfinder.utils.readToken
+
+private const val DUMP_DIR = "repos/results"
+private const val URLS_PATH = "repos/urls.json"
+private const val CONFIG_PATH = "repos/config.json"
+private const val TOKEN_PATH = "repos/token.txt"
```

Comment:
I suggest using CLI args for passing paths to input, output, and config.

Reply:
Or, I think is better to use only one config. Users can specify all paths, tokens in a single JSON and pass it to the tool. Btw, let's store template config in the root folder.

Reply:
fixed